paasta_tools/check_services_replication_tools.py

```diff
@@ -190,14 +208,24 @@ def main(
     cluster = system_paasta_config.get_cluster()
     replication_checker: ReplicationChecker
 
-    if mesos:
-        tasks_or_pods, slaves = get_mesos_tasks_and_slaves(system_paasta_config)
-        replication_checker = MesosSmartstackEnvoyReplicationChecker(
-            mesos_slaves=slaves,
+    # throw an exception if kubernetes_services flag is false and namespace is not set
+    if kubernetes_services is False and namespace is None:
+        raise Exception("Please set namespace when calling the main function")
+
+    if kubernetes_services:
```

Comment:
yep, ``kubernetes_services`` is a flag that is true when being called by ``check_kubernetes_services_replication`` and false otherwise

Reply:
thank you, will go with the first solution of removing the flag and just checking if namespace is none since the second one will require us to change the cron job (I assume in puppet)